cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
This is so we don't hard-code this in all our tests. To navigate to a page in Cypress, the `cy.visit(someUrl)` is used. If we write `cy.visist('/new')`, Cypress will no to navigate to `http://localhost:3000/new`.